test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -380,6 +398,101 @@ private static void testCheckCastRefClassOnNull(ClassWriter cw, String className
 		mv.visitEnd();
 	}
 
+	private static void testUnresolvedValueTypeDefaultValue(ClassWriter cw, String className, String valueUsedInCode, String[] fields) {
```

Comment:
`fields` is not used in `testUnresolvedValueTypeDefaultValue`, why is it passed in?

Reply:
Almost all of the `test` methods have a `fields` argument, even though most of them don't use it, so I just followed suit.  I would be happy to drop the `fields`argument in this case.

Reply:
Agreed, no need to add `field` params if they are not being used. This was probably an oversight in the other tests.

Reply:
Removed the `fields` argument in commit 57a652d00483cac5c92f341e93509a40bd5516f2.